lib/player_manager.rb

```diff
@@ -1,23 +1,31 @@
-require 'game_state'
+require 'ai_basic'
```

Comment:
I don't see `AiBasic` or `AiMinimax` being used anywhere in this class - are the requires required?